Tests/test_SffIO.py

```diff
@@ -561,9 +561,8 @@ def fileiter(handle):
     try:
         print(ReadRocheXmlManifest(
               open("Roche/E3MFGYR02_alt_index_at_end.sff", "rb")))
-        assert False, "Should fail!"
     except ValueError:
-        pass
+        raise ValueError("Should fail.")
```

Comment:
I think you've accidentally flipped the meaning of the test. This looks like another case were we could use ``self.assertRaises(ValueError)``?

Reply:
As this wasn't a part of a unittest TestCase, I've called the function directly instead of accessing via a reference to self. Not sure if it was the best way to go about it, or if a new TestCase would be more appropriate.

Reply:
You're absolutely right - I didn't see that from the context of the diff. Does this new form work nicely (I've never used it that way myself).

Reply:
As far as I'm aware, it works equivalently and it appeared to work okay locally on my machine.